### PR TITLE
Add VineBlockElement

### PR DESCRIPTION
**Pull request by nitro-marky**
## What does this change?

Implements Vine embeds (I live to Vine). 

Rather brillitantly at the time this was added the Vine archive was down, so it looks broken at the moment but it's like that on PROD too! It is hidden behind `ClickToView`.

It is also hidden by a `clickToView` for consent reasons, and to hide the broken parts straight away.

Before:

Screenshot 2021-02-19 at 15 39 10

After:

Screenshot 2021-02-22 at 17 09 05

## Why?

no-answer

### CODE REVIEW

**Comment by nitro-marky** [comment]
[VineBlockElement](https://trello.com/c/qDfA3hHU/1772-vineblockelement)

File: src/web/components/elements/VineBlockComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	element: VineBlockElement;
+}> = ({ element }) => {
+	return (
+		<>
+			{element.url && element.width && element.height && (
+				<MaintainAspectRatio
+					height={element.height}
+					width={element.width}
+				>
+					<div className="element-vine">
```
Comment (oliverlloyd) [inline src/web/components/elements/VineBlockComponent.tsx:14]: I guess we can remove this className?

File: src/web/lib/ArticleRenderer.tsx [hunk 2]
```diff
@@ -479,6 +480,8 @@ export const ArticleRenderer: React.FC<{
 							<TweetBlockComponent element={element} />
 						</Figure>
 					);
+				case 'model.dotcomrendering.pageElements.VineBlockElement':
+					return <VineBlockComponent element={element} />;
```
Comment (oliverlloyd) [inline src/web/lib/ArticleRenderer.tsx:484]: This should have `ClickToView` around it, for the legals

**Review by oliverlloyd — APPROVED** [review]
youths

**Comment by shtukas** [comment]
Superseded by: https://github.com/guardian/dotcom-rendering/pull/2607

**Pull request closed without merging**